persistence/sql/migrations/templates/20210307130559_courier_message_template.up.fizz

```diff
@@ -0,0 +1,2 @@
+add_column("courier_messages", "template_type", "string", {default: ""})
```

Comment:
I believe this will fail if `courier_messages` has rows. The best way to find out is to run `https://github.com/ory/kratos/blob/master/Makefile#L136` and then update https://github.com/ory/kratos/tree/master/persistence/sql/migratest/testdata and then running https://github.com/ory/kratos/blob/master/persistence/sql/migratest/migration_test.go or the refresh script.

Reply:
ok, thanks for the heads up, will give this a try.

Reply:
What do you mean exactly by `update https://github.com/ory/kratos/tree/master/persistence/sql/migratest/testdata`?

Reply:
@aeneasr hey, left a question here

Reply:
Ah sorry, I missed it:

1. Run `make migrations-sync`
2. A new file appears in https://github.com/ory/kratos/tree/master/persistence/sql/migratest/testdata
3. Add data to the new file, analogous to e.g. https://github.com/ory/kratos/blob/master/persistence/sql/migratest/testdata/20200831110752_testdata.sql - e.g. a new courier message
4. Run the tests
5. You might need to update some of the expected outputs which you can do with `make migratest-refresh`

Reply:
hmm, i get the following error when running `make migrations-sync`

```
❯ make migrations-sync
go build -o .bin/cli -tags sqlite github.com/ory/cli
# github.com/ory/cli/cmd/dev/newsletter
../../../../../go/pkg/mod/github.com/ory/cli@v0.0.41/cmd/dev/newsletter/pkg.go:86:12: undefined: httpx.NewResilientClientLatencyToleranceMedium
make: *** [.bin/cli] Error 2
```

Reply:
bumping ory/cli version to v0.0.48 fixes this, continuing

Reply:
@aeneasr tests passed! added the new migratest file

Reply:
looking into getting e2e tests working now. having trouble running them locally though as mentioned on #1149. some issue with the record key not being present

Reply:
Did you figure this out?

Reply:
yep, figured the e2e tests out, just need to remove the --record flag locally.
and I followed the process you specified for migrations and everything passed.